com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/utils/FileSystemService.java

```diff
@@ -73,4 +74,15 @@
    */
   @Nonnull
   String decodeURI(@Nonnull String uri);
+
+  /**
+   * This method validate an URI to verify if is defined as extended document, accordining to a
+   * regex.
```

Comment:
A couple of typos here, "This method validates the URI to verify if is defined as an extended document, according to the regex."